YoCode/LoadingAnimation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace YoCode
+{
+    public class LoadingAnimation
+    {
+        static string dots;
+        public static void RunLoading()
+        {
+            var cursorPos = Console.CursorTop;
+            Console.WriteLine(messages.Fireplace);
+            Console.WriteLine(messages.ParagraphDivider);
+            Console.WriteLine($"Get comfortable, YoCode is about to finish gathering{Environment.NewLine}your results");
```

Comment:
Instead of "YoCode is about to finish gathering" I would just says "YoCode is gathering", it's easier to read.